# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../daemon/cross.patch
# Copyright (C) 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- daemon-0.6.3/rules.mk.vanilla	2007-06-17 13:15:30.792904634 +0200
+++ daemon-0.6.3/rules.mk	2007-06-17 13:15:42.365564123 +0200
@@ -45,7 +45,6 @@
 install-daemon-bin:
 	mkdir -p $(APP_INSDIR)
 	install -m 755 $(DAEMON_TARGET) $(APP_INSDIR)
-	strip $(patsubst %, $(APP_INSDIR)/%, $(notdir $(DAEMON_TARGET)))
 
 install-daemon-man: man-daemon
 	@mkdir -p $(APP_MANDIR); \
